### DISCUSSION TITLE
[Feature Request] Allow mobile friendly page (basically second homepage setup just for mobile)

### DISCUSSION THREAD

**Discussion by Joshndroid** [linuxserver/Heimdall#670] [Ideas]
Great program.

I was wondering if there was an ability to create a second setup, basically entirely for mobile devices.
I find that the current setup, while awesome and handy, has huge icons and mainly catered towards a desktop environment.
I find myself needing to scroll to access various things that are not part of the immediate few that show at the top.
Also, at present I have a number of items on the start page that I use mainly on the desktop side, but have no real use for on mobile due to having 'an app for that.'
I think it would be cool to have the ability to have a separate desktop and mobile experiences, handling all within the one setup.
I have VPN always on connection to home so could basically take over multiple aspects of my experience whether at home or not with this page.

In reality I am not sure if it would be even possible but a couple of avenues could be;
 - Maybe use a different port number for the setup of a mobile friendly page
 - Utilise the user agent string to trigger a 'mobile page'
 - Differentiate setups into a 'desktop' and 'mobile' state.
 - Allow for the pasting of app specific launching strings for a program (not really sure if this would be much of a benefit tho tbh)

**Comment by xekil** [comment]
+ 10000 At least it would be nice to adapt Heimdall to a smartphone screen to have two shortcuts per line rather than one!
